VocaDb.ResXFileCodeGenerator/StringBuilderGenerator.cs

```diff
@@ -32,7 +35,7 @@ public string Generate(Stream resxStream, GeneratorOptions options)
 
 		builder.Append("    ");
 		builder.Append(options.PublicClass ? "public" : "internal");
-		builder.Append(" static class ");
+		builder.Append(options.StaticClass ? " static class " : " class ");
```

Comment:
Does it make sense to make not only classes, but also properties, non-static?

Reply:
The properties are static regardless, to enable "old-school" way of accessing with `MyResource.MyLocalizedString`.